Workade/Share/CustomSegmentedControl.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CustopSegmentController.swift
+//  Workade
+//
+//  Created by Hong jeongmin on 2022/10/20.
+//
+
+import UIKit
+
+class CustomSegmentedControl: UISegmentedControl {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.setupSegmentControl()
+    }
+    
+    override init(items: [Any]?) {
+        super.init(items: items)
+        self.setupSegmentControl()
+    }
```

Comment:
```suggestion
    override init(frame: CGRect) {
        super.init(frame: frame)
        self.setupSegmentedControl()
    }
    
    override init(items: [Any]?) {
        super.init(items: items)
        self.setupSegmentedControl()
    }
```
[P3] 이것도 수정되면 좋지 않을까요??

Reply:
두가지 사항 모두 적용하여 반영하였습니다 :) 👍